GameData/Kerbalism/Settings.cfg

```diff
@@ -3,8 +3,6 @@ Kerbalism
   // profile used
   Profile = default                   // valid values are: 'default', 'classic, 'none' or custom profiles
 
-  CrewMass = 0.09375
```

Comment:
Is there a reason this can not stay a configurable value? Mods like realism overhaul may want to update this with a number based on human weights.

Reply:
I wanted to use a value from PhysicsGlobal, but that seems to be zero.

it won’t stay that way it is, it’s work in progress. but rest assured i’m not going to hardcode any values.

Reply:
Absolutely understood that this branch is a WIP, just wanted to chime in before anything got built around this that could make it harder to change later.